proto/destination.proto

```diff
@@ -166,6 +172,47 @@ message ProtocolHint {
   }
 }
 
+// Configures the parameters used to initialize an HTTP/2 connection.
+message Http2ClientParams {
+  // Overrides the default client flow control settings.
+  FlowControl flow_control = 1;
+
+  // Enables keep-alive timeouts.
+  KeepAlive keep_alive = 2;
+
+  // Configures Hyper internals.
+  Internals internals = 3;
+
+  message FlowControl {
```

Comment:
is the idea that the control plane will always provide these values? or can the control plane leave them unspecified and allow the proxy to use some defaults?

Reply:
If the control plane omits them, the proxy will use defaults. I don't have immediate plans to set these values from the control plane.